readyapi4j-core/src/main/java/com/smartbear/readyapi4j/AssertionNames.java

```diff
@@ -0,0 +1,19 @@
+package com.smartbear.readyapi4j;
+
+class AssertionNames {
+    static final String VALID_HTTP_STATUS_CODES = "Valid HTTP Status Codes";
+    static final String INVALID_HTTP_STATUS_CODES = "Invalid HTTP Status Codes";
+    static final String SIMPLE_CONTAINS = "Contains";
+    static final String SIMPLE_NOT_CONTAINS = "Not Contains";
+    static final String X_PATH_MATCH = "XPath Match";
```

Comment:
being picky here - but I would skip the underscore after the X -> XPATH_MATCH and XQUERY_MATCH